sup3r/bias/bias_transforms.py

```diff
@@ -64,6 +65,12 @@ def local_linear_bc(input, feature_name, bias_fp, lr_padded_slice,
         source shape will be used.
     out_range : None | tuple
         Option to set floor/ceiling values on the output data.
+    smoothing : float
```

Comment:
why did you add this to the transforms module? Just so we don't have to recalc the bc factors? Or so you can test a bunch of smoothing factors at runtime? 

The docstring isn't accurate: no inside/outside spatial domain or threshold input here

Reply:
Yeah so we don't have to recalc, which also allows me to test. Wasn't sure if we should just keep the smoothing here and remove the other.

Reply:
One argument for only keeping the smoothing at the bias calc step: we'll be less likely to double-smooth. Consider that we will probably smooth the extrapolated extent and will be hard to not duplicate the smoothing in the transforms module. 

For testing i would advocate for just writing a script outside of the repo that duplicates the bias correction files with different smoothing factors.

Reply:
Agree on the double smoothing, although I'm not sure how much this matters outside the valid domain. I would argue for keeping the smoothing in the transforms since I don't see a reason to have to rerun bias-calc just to change smoothing. Keeping the stored bias correction factors as "true" as possible seems reasonable.

Reply:
Yeah you just loose the ability to smooth inside the valid domain vs. outside... That's fine though if that's your preference... I'd say keep both because the ability to smooth inside vs. outside is important esp. for sup3rcc